README.md

```diff
@@ -31,7 +31,9 @@
 
 </div>
 
-# Starlite
+# Starlite → Litestar
+
+## [We are renaming to Litestar](https://twitter.com/StarliteAPI/status/1642958305231265792?s=20)
```

Comment:
We should probably link to something that gives more context about the decision?

Reply:
Maybe this should be [the post on r/python](https://www.reddit.com/r/Python/comments/129n4dc/renaming_starlite_to_litestar/) or an even more detailed post on our own subreddit?

Reply:
Added reddit post that has more context